SetupBLT.cmake

```diff
@@ -51,13 +51,21 @@ if (NOT BLT_LOADED)
     include(${BLT_ROOT_DIR}/cmake/BLTOptions.cmake)
 
     ################################
-    # CMP0057 New is required by blt_setup_target()
-    # in order to support the IN_LIST if() operator
+    # CMake Policies
     ################################
+    # Support IN_LIST operator for if()
```

Comment:
What version was this added? Might be worth noting, like you have in the comment below.